.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/src/config
```

Comment:
![image](https://user-images.githubusercontent.com/4306/69003310-4dc34600-08b4-11ea-8c3c-644fef82bc64.png)

See the red icons at the end of the line? This is GitHub complaining that the file doesn't end with a newline (`\n`). **No need to fix this,** but just be aware that in some systems/languages, that extra return can cause unintended errors. It's a good habit to be in to always add one last return.